lib/elm-test.js

```diff
@@ -505,7 +505,7 @@ function generatePackageJson(filePathArgs) {
 function ensurePackagesInstalled(dir) {
   return new Promise(function(resolve, reject) {
     dns.lookup("elm-lang.org", function(err) {
-      if (err && err.code == "ENOTFOUND") {
+      if (err && (err.code == "ENOTFOUND" || error.code == "EAI_AGAIN")) {
```

Comment:
Should be `err.code` (`error` is not defined).

Reply:
Thanks! So embarrassing, I'm so happy there is a review process.